lib/srv/alpnproxy/auth/auth_proxy_test.go

```diff
@@ -22,55 +22,41 @@ import (
 	"testing"
 	"time"
 
-	"github.com/gravitational/teleport/api/types"
-	"github.com/gravitational/teleport/lib/services"
 	"github.com/stretchr/testify/require"
 )
 
-type mockAuthGetter struct {
-	servers []types.Server
-}
-
-func (m mockAuthGetter) GetClusterName(...services.MarshalOption) (types.ClusterName, error) {
-	return nil, nil
-}
-
-func (m mockAuthGetter) GetAuthServers() ([]types.Server, error) {
-	return m.servers, nil
-}
-
 func TestDialLocalAuthServerNoServers(t *testing.T) {
-	s := NewAuthProxyDialerService(nil, mockAuthGetter{servers: []types.Server{}})
+	s := NewAuthProxyDialerService(nil, "", []string{})
```

Comment:
nit: can we get some inline comments so the meaning is clearer?

```suggestion
	s := NewAuthProxyDialerService(nil /* foo */, "" /* bar */, nil  /* authServers */)
```